cfme/tests/cloud_infra_common/test_custom_attributes_rest.py

```diff
@@ -0,0 +1,298 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+import fauxfactory
+
+from cfme import test_requirements
+from cfme.cloud.provider import CloudProvider
+from cfme.common.vm import VM
+from cfme.infrastructure.provider import CloudInfraProvider, InfraProvider
+from cfme.utils import error, testgen
+from cfme.utils.blockers import BZ
+from cfme.utils.generators import random_vm_name
+from cfme.utils.log import logger
+from cfme.utils.providers import ProviderFilter
+from cfme.utils.rest import assert_response, delete_resources_from_collection
+from cfme.utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.long_running,
+    pytest.mark.tier(2),
+    test_requirements.rest
+]
+
+
+pytest_generate_tests = testgen.generate(
```

Comment:
**Optional**: This can be replaced with: pytest.mark.provider([CloudInfraProvider], scope="module") in pytestmark list